BUILD.bazel

```diff
@@ -757,6 +756,26 @@ ray_cc_library(
     ],
 )
 
+# PubSub RPC module
+ray_cc_library(
+    name = "pubsub_rpc",
+    deps = [
+        ":pubsub_cc_grpc",
+        ":grpc_common_lib",
```

Comment:
Could you please leave a TODO under core (i.e. `TODO(core)`) so we could revisit after `grpc_common_lib` is broken down into smaller targets?